src/main/resources/migration/common/V44__add_comparison_discrepancy_tables.sql

```diff
@@ -0,0 +1,46 @@
+create table comparison_person_discrepancy_impact
+(
+    id serial constraint discrepancy_impact_pk primary key,
+    impact varchar(30) not null
+);
+
+create table comparison_person_discrepancy_priority
+(
+    id serial constraint discrepancy_priority_pk primary key,
+    priority varchar(30) not null
+);
+
+create table comparison_person_discrepancy
+(
+    id serial constraint comparison_person_discrepancy_pk primary key,
+    comparison_person_id integer not null,
+    impact_id integer not null,
+    priority_id integer not null,
+    detail text,
+    action text,
+    created_by varchar(20) not null,
```

Comment:
I just raised a ticket https://dsdmoj.atlassian.net/browse/CRS-1794 as I noticed that fields capturing username are a bit all over the place in their length.

Reply:
ok, I've updated to 40 for now but can update again when someone has a look at that ticket